projects/kit/components/input-files/input-files.directive.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, ElementRef, HostBinding, Inject} from '@angular/core';
+import {TuiIdService} from '@taiga-ui/cdk';
+
+@Component({
+    selector: `input[tuiInputFiles]`,
+    template: ``,
+})
+export class TuiInputFilesDirective {
```

Comment:
This directive should probably also rely on default options token.

Reply:
fixed